site_root/input/resources/profiles/StructureDefinition-kfdrc-research-study.json

```diff
@@ -0,0 +1,79 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-research-study",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-research-study",
+  "version": "0.1.0",
+  "name": "kfdrc-research-study",
+  "title": "Kids First DRC ResearchStudy",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "ResearchStudy",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/ResearchStudy",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "ResearchStudy",
+        "path": "ResearchStudy"
+      },
+      {
+        "id": "ResearchStudy.extension",
+        "path": "ResearchStudy.extension",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "url"
+            }
+          ],
+          "ordered": false,
+          "rules": "open"
+        }
+      },
+      {
+        "id": "ResearchStudy.extension:display-name",
+        "path": "ResearchStudy.extension",
+        "sliceName": "display-name",
+        "sliceIsConstraining": false,
+        "label": "display_name",
+        "short": "Display name of the Kids First DRC ResearchStudy",
+        "definition": "Display name (shortened title) of the Kids First DRC ResearchStudy.",
+        "min": 0,
+        "max": "1",
+        "type": [
+          {
+            "code": "Extension",
+            "profile": [
+              "http://fhir.kids-first.io/StructureDefinition/display-name"
+            ]
+          }
+        ],
+        "mustSupport": true,
```

Comment:
why do these both have `true` for `mustSupport`?

Reply:
`"mustSupport"` is about a server understanding the meaning of an extension. `"min": 0` (or 1) means there has to be a value but the server still doesn't enforce that. If the server doesn't understand `"mustSupport"`, it shouldn't accept the resource.